src/Sentry/Internal/Hub.cs

```diff
@@ -553,6 +553,12 @@ public void CaptureFeedback(SentryFeedback feedback, Action<Scope> configureScop
 
         try
         {
+            if (!EmailValidator.IsValidEmail(feedback.ContactEmail))
+            {
+                _options.LogWarning("Feedback dropped due to invalid email format: '{0}'", feedback.ContactEmail);
+                return;
```

Comment:
An alternative here might be to set the email to an empty string (since the email address isn't strictly required). That way SDK users wouldn't lose potentially valuable feedback if their users make a typo when entering their email address.